### ISSUE TITLE
The relationship between neural network depth and accuracy

### ISSUE THREAD

**Issue by williamli186** [tensorflow/tensorflow#15609]
Hi:
I am learning to design a simple neural network, and try to identify 28x28 pixel greyscale images in the MNIST dataset.
I find that the neural network depth is not directly proportional to the recognition rate,

one layer neural network recognition rate: 92%
two layer neural network recognition rate: 94%
four layer neural network recognition rate: 91.8%

Can someone help me analyze it?
![6](https://user-images.githubusercontent.com/31270354/34325172-85768d48-e8c5-11e7-831a-6d11b9b02bf5.PNG)
(Please click on the picture to see the complete picture information)

**thank you very much!**

**Comment by bignamehyp** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
